src/order/compactly_generated.lean

```diff
@@ -120,6 +123,25 @@ begin
     use t, exact ⟨htS, by rwa ←htsup⟩, },
 end
 
+/-- A compact element has the property that any directed set lying strictly below k has
+its Sup strictly below k -/
+lemma Sup_lt_of_directed_set_lt_compact (k : α) (hk : is_compact_element k) (hbot : k ≠ ⊥) :
```

Comment:
```suggestion
lemma is_compact_element.directed_Sup_lt_of_lt {k : α} (hk : is_compact_element k) (hbot : k ≠ ⊥) :
```
The variable `k` should probably be implicit. Also, for this to work, you either need `k ≠ ⊥` or `s.nonempty`, and my guess is that the latter is a better assumption to work with, if only out of parallelism with `le_of_directed_Sup_le`. I've also suggested a name that I think is more analogous to that other lemma.